src/main/groovy/com/ullink/gradle/opencover/OpenCoverNUnitPlugin.groovy

```diff
@@ -17,8 +17,17 @@ class OpenCoverNUnitPlugin implements Plugin<Project> {
         task.doFirst {
             nunit.prepareExecute()
         }
+
         task.conventionMapping.map 'targetExec', { nunit.nunitExec }
         task.conventionMapping.map 'targetExecArgs', { nunit.commandArgs }
+        task.conventionMapping.map 'parallelForks',{ nunit.parallelForks }
+
+        def intermediateNunitResultsPath = new File(nunit.getReportFolderImpl().toString() , "intemediate-results-nunit")
+
+        task.conventionMapping.map 'intermediateNunitResultsPath', { intermediateNunitResultsPath}
+        task.conventionMapping.map 'parallelTargetExecArgs', {
+            nunit.getTestInputAsList(nunit.where?.value).collect({nunit.buildCommandArgs(it,  new File (intermediateNunitResultsPath,  UUID.randomUUID().toString() + ".xml"))})}
```

Comment:
can create the folder in this callback, it'll be called during execution when getting the value

Reply:
Ok, it means we don't need anymore to map the *intermediateNunitResultsPath*